nickytonline/astro-partykit-starter
# feat: adding storybook to starter kit

## Description
This PR adds Storybook to the starter kit. I have added stories for both the `Party` and `Modal` components. I have also added the `@storybook/addon-a11y` addon and a detailed description on how to use Storybook in the `README`. 

## What type of PR is this? (check all applicable)


## Related Tickets & Documents
closes #54 


## Mobile & Desktop Screenshots/Recordings

<img width="1287" alt="Screenshot 2024-01-27 at 10 37 19 PM" src="https://github.com/nickytonline/astro-partykit-starter/assets/67210629/1addcc9c-de89-4652-9547-8e1a29bb1e2b">

<img width="1257" alt="Screenshot 2024-01-27 at 10 37 42 PM" src="https://github.com/nickytonline/astro-partykit-starter/assets/67210629/a3b35a9f-568a-4026-8789-c61b28078f05">


## Steps to QA

1. Pull down this branch locally
2. Install the new dependencies
3. Run `npm run storybook`
4. see the new Party and Modal components on `http://localhost:6006/`


## Added to documentation?


## [optional] Are there any post-deployment tasks we need to perform?



## [optional] What gif best describes this PR or how it makes you feel?
